### PR TITLE
add more metrics

### PR DESCRIPTION
**Pull request by pwinckles**
**JIRA Ticket**: https://jira.lyrasis.org/browse/FCREPO-3443

# What does this Pull Request do?

Adds some metrics around OCFL and DB operations.

# How should this be tested?

Follow the [steps here](https://wiki.lyrasis.org/display/FEDORA6x/Metrics) to capture metrics. The dashboard at the bottom of the page has some example graphs with the new metrics.

# Interested parties
@fcrepo/committers

### CODE REVIEW

File: fcrepo-auth-webac/src/main/java/org/fcrepo/auth/webac/WebACAuthorizingRealm.java [hunk 1]
```diff
@@ -87,7 +89,8 @@
     @Inject
     private ResourceFactory resourceFactory;
 
-    @Inject
+    @Autowired
```
Comment (awoods) [inline fcrepo-auth-webac/src/main/java/org/fcrepo/auth/webac/WebACAuthorizingRealm.java:92]: Why did this (and other examples) need to change from `@Inject` to `@Autowired`?

Comment (pwinckles) [reply fcrepo-auth-webac/src/main/java/org/fcrepo/auth/webac/WebACAuthorizingRealm.java:92]: It seems like Spring does not like resolving the `@Qualifier` for `@Inject`, at least in some circumstances.

File: fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/AbstractPersister.java [hunk 2]
```diff
@@ -56,14 +56,14 @@
      * to determine which operations the persister knows how to handle.
      */
     private final ResourceOperationType resourceOperationType;
-    protected final FedoraToOcflObjectIndex index;
+    protected final FedoraToOcflObjectIndex oclfIndex;
```
Comment (awoods) [inline fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/AbstractPersister.java:59]: typo: "oclf"

File: fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/CreateNonRdfSourcePersister.java [hunk 3]
```diff
@@ -53,6 +53,6 @@ public void persist(final OcflPersistentStorageSession session, final ResourceOp
         final String ocflId = mapToOcflId(session.getId(), rootObjectId);
         final OcflObjectSession ocflObjectSession = session.findOrCreateSession(ocflId);
         persistNonRDFSource(operation, ocflObjectSession, rootObjectId.asBaseId());
-        index.addMapping(session.getId(), resourceId.asResourceId(), rootObjectId.asBaseId(), ocflId);
+        oclfIndex.addMapping(session.getId(), resourceId.asResourceId(), rootObjectId.asBaseId(), ocflId);
```
Comment (awoods) [inline fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/CreateNonRdfSourcePersister.java:56]: typo

File: fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/CreateRdfSourcePersister.java [hunk 4]
```diff
@@ -73,6 +73,6 @@ public void persist(final OcflPersistentStorageSession session, final ResourceOp
         final String ocflObjectId = mapToOcflId(session.getId(), rootObjectId);
         final OcflObjectSession ocflObjectSession = session.findOrCreateSession(ocflObjectId);
         persistRDF(ocflObjectSession, operation, rootObjectId.asBaseId());
-        index.addMapping(session.getId(), resourceId.asResourceId(), rootObjectId.asBaseId(), ocflObjectId);
+        oclfIndex.addMapping(session.getId(), resourceId.asResourceId(), rootObjectId.asBaseId(), ocflObjectId);
```
Comment (awoods) [inline fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/CreateRdfSourcePersister.java:76]: typo

File: fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/DeleteResourcePersister.java [hunk 5]
```diff
@@ -62,7 +62,7 @@ public void persist(final OcflPersistentStorageSession session, final ResourceOp
         }
 
         if (!objectSession.containsResource(resourceId.getResourceId())) {
-            index.removeMapping(session.getId(), resourceId);
+            oclfIndex.removeMapping(session.getId(), resourceId);
```
Comment (awoods) [inline fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/DeleteResourcePersister.java:65]: typo

**Review by awoods — CHANGES_REQUESTED** [review]

**Comment by pwinckles** [comment]
Typos fixed!

**Review by awoods — APPROVED** [review]

**Pull request merged**